### ISSUE TITLE
What were your hyperparameter?

### ISSUE THREAD

**Issue by seongq** [GeWu-Lab/OGM-GE_CVPR2022#30]
Hello.
I want know abot your setting for emotion recognition with CREMA-D.
I have tested your code for my research. 
In your paper, your accuracy applied OGM-GE was 62.2 when dataset is CREMA-D and fusion method is sum.
I have tested with the same setting. But,  my best acc was 61.

What were your hyperparameters for emotion recognition? (alpha, batchsize and learning rate)

**Comment by seongq** [comment]
Your parameter can be found on this github page.
I did not read closely. 
thank you
